LuizWeitz.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "@LuizWeitz"
+project: "E-reader using Raspberry Pi Pico"
+---
+
+# E-reader With Raspberry Pi Pico
+
+## Summary
+
+E-reader using Raspberry Pi Pico with E-Ink technology screen with rechargeable battery that fits in the palm of the hand, through which I intend to expand my adventures in the world of reading.
+
+## Plan
+
+The first step is to plan how the components will be organized. <br>
+The second step is to connect all components through wire and solder.  <br>
+The third step is to plan a simple case in 3d and print out. <br>
+The fourth step is to create a simple software that reads the books and sends them to the screen with the possibility of changing pages.  <br>
+The fifth step is to test everything.  <br>
+
+### one of the prototypes:
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+|  Raspberry Pi Pico Board | https://www.aliexpress.com/item/1005002890682924.html?spm=a2g0o.cart.0.0.267538daPxA5wQ&mp=1 | $9.58 |
+| Waveshare 7.5inch E-Paper E-Ink Display Module For Raspberry Pi Pico | https://www.aliexpress.com/item/1005002841576759.html?spm=a2g0o.cart.0.0.267538daPxA5wQ&mp=1 | $63.89 |
+| MicroSD Card Adapter| https://www.aliexpress.com/item/1005001309671718.html?spm=a2g0o.cart.0.0.267538daPxA5wQ&mp=1 | $6.37 |
+| Lexar Card SD 32GB | https://www.aliexpress.com/item/1005003780732823.html?spm=a2g0o.cart.0.0.267538daPxA5wQ&mp=1 | $6.04 |
+| Raspberry Pi Zero UPS-Lite Power Supply Board with 1000mAh 3.7V | https://www.aliexpress.com/item/1005003792770003.html?spm=a2g0o.cart.0.0.267538daPxA5wQ&mp=1 | $24.24 |
+| K7 X1 3D Printer Mini | https://www.aliexpress.com/item/1005002668227870.html?spm=a2g0o.cart.0.0.5b8338daT6TgTX&mp=1 | $105.01 |
```

Comment:
Do you already have fillament for your printer? I don't see any filament in your budget

Reply:
Hi, yes, I recently gained almost 1 kg of black pla filament, but since I don't have a printer, I left it at a standstill 😕 The good thing about pla is that it is an easy to print and biodegradable material.